### PR TITLE
[composer] attachment issues

### PR DESCRIPTION
**Pull request by heathergh**
# Code changes

- [x] The 'x' to remove the attachment from the message is small

- [x] Improved attachment error handling (file name and size are hidden if there's an error)

- [x] Loading spinner when uploading attachment is small

- [x] Whole filename + file extension are visible

# Readiness checklist

- [x] Cypress tests passing?
- [ ] New cypress tests added?
- [ ] Included before/after screenshots, if the change is visual

# License

I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: components/composer/src/index.html [hunk 1]
```diff
@@ -57,14 +58,7 @@
...
   <body>
-    <nylas-composer
-      id="demo-composer"
-      theme="dark"
-      template="Hey what up!<br/>
-      <br/>
-      <br/>
-      Thanks,
-      -Phil">
+    <nylas-composer id="ba4f94f7-4155-4f0d-9b34-c6b8d4e3f344" theme="dark">
```
Comment (krisgardiner) [inline components/composer/src/index.html:61]: Was this left by accident?

File: components/composer/src/components/Attachment.svelte [hunk 2]
```diff
@@ -79,31 +76,38 @@
...
       <div class="file-info__right">
         {#if attachment.loading}
-          <LoadingIcon class="LoadingIcon" />
+          <LoadingIcon
+            style="fill: var(--composer-icons-color, #666774); width: 15px; height: 15px; animation: rotate 0.5s infinite linear;"
+          />
+        {/if}
+        {#if attachment.error}
+          <span class="file-info__error"
+            >Error: Please try attaching the file again.</span
```
Comment (krisgardiner) [inline components/composer/src/components/Attachment.svelte:108]: Oh no! The closing tag bracket issue thing resurfaced 😞

**Review by krisgardiner — APPROVED** [review]

File: components/composer/src/index.html (hunk 1 above)
Comment (heathergh) [reply components/composer/src/index.html:61]: yes 😅

File: components/composer/src/components/Attachment.svelte (hunk 2 above)
Comment (heathergh) [reply components/composer/src/components/Attachment.svelte:108]: :<

**Pull request merged**